dali/ft/filecopy.cpp

```diff
@@ -1224,6 +1224,8 @@ void FileSprayer::calculateSprayPartition()
                 distributedTarget->setECL(recStru.str());
         }
     }
+    if (compressedInput && compressOutput && streq(encryptKey.str(),decryptKey.str()))
```

Comment:
I assume encryptKey corresponds to the compressed output? and decryptKey, corresponds to the compressed input ?
Couldn't they differ?
Couldn't the output be compressed, but the input not?